### DISCUSSION TITLE
Can't get `hostname` in ssh.conf to work

### DISCUSSION THREAD

**Discussion by rabbiveesh** [kovidgoyal/kitty#6612] [Q&A]
I would like to set up the ssh kitten with settings specific to one user on a particular server. However, I can't get the `hostname` line to work correctly in my `ssh.conf`.

Let's say the user is `veesh` and the server is `prod`.

using `hostname veesh@prod` does not work. In testing, using `hostname prod` isn't working either. however `hostname veesh@*` does work.

What am I missing here? From the docs, this looks like it shoudl be straightforward

**Comment by kovidgoyal** [comment]
Read #6609
